### PR TITLE
Add msg where missing

### PR DESCRIPTION
**Pull request by francescopellegrini**
Also use new wording on Omar

### CODE REVIEW

File: src/clj/game/cards/ice.clj [hunk 1]
```diff
@@ -2466,15 +2466,16 @@
...
-                                               (end-run state :runner eid card)))
-                                 (do (system-msg state :runner "adds Meridian to their score area as an agenda worth -1 agenda points")
-                                     (as-agenda state :runner card -1)
+                  :effect (req (if (str/starts-with? target "Corp")
+                                 (wait-for (gain-credits state :corp 4)
+                                           (end-run state :runner eid card))
+                                 (do (as-agenda state :runner card -1)
                                      (when current-ice
                                        (encounter-ends state side eid))
                                      (effect-completed state side eid))))}]})
```
Comment (NoahTheDuke) [inline src/clj/game/cards/ice.clj:2481]: Can't use the eid and then complete it. Gotta be an if, not when.

File: src/clj/game/cards/identities.clj [hunk 2]
```diff
@@ -1521,12 +1521,12 @@
...
              :req (req (and (get-in card [:special :omar-run])
                             (= :archives (-> run :server first))))
-             :prompt "Treat as a successful run on which server?"
+             :prompt "Choose one"
              :choices ["HQ" "R&D"]
+             :msg (msg "change the attacked server to " target)
              :effect (req (let [target-server (if (= target "HQ") :hq :rd)]
                             (swap! state assoc-in [:run :server] [target-server])
-                            (trigger-event state :corp :no-action)
-                            (system-msg state side (str "uses Omar Keung: Conspiracy Theorist to make a successful run on " target))))}
```
Comment (NoahTheDuke) [inline src/clj/game/cards/identities.clj:1529]: Why remove this?

**Review by NoahTheDuke — CHANGES_REQUESTED** [review]

File: src/clj/game/cards/identities.clj (hunk 2 above)
Comment (francescopellegrini) [reply src/clj/game/cards/identities.clj:1529]: I moved it to `:msg`

File: src/clj/game/cards/ice.clj (hunk 1 above)
Comment (francescopellegrini) [reply src/clj/game/cards/ice.clj:2481]: cc @NBKelly (see #6652)

Comment (francescopellegrini) [reply src/clj/game/cards/ice.clj:2481]: Indeed I just tested it locally and it creates an infinite loop of encounter and approach prompts

**Pull request merged**